common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -100,7 +100,8 @@ class WhiskConfig(
     val dbPort = this(WhiskConfig.dbPort)
     val dbWhisk = this(WhiskConfig.dbWhisk)
     val dbAuths = this(WhiskConfig.dbAuths)
-    val dbActivations = this(WhiskConfig.dbActivations)
+    lazy val dbSplitActionsAndActivations = this(WhiskConfig.dbSplitActionsAndActivations).toBoolean
```

Comment:
i'm having a hard time seeing how this is used - or can work if the value is false.
convince me.